tests/FakeItEasy.Specs/SelfInitializedFakesSpecs.cs

```diff
@@ -142,12 +436,13 @@ public static class SelfInitializedFakesSpecs
 
                         countDuringPlayback = playbackFakeService.GetCount("8");
                     }
-                });
+                })
+                .Teardown(() => File.Delete(fileRecorderPath));
 
-            "Then the recording fake returns the wrapped service's result"
+            "Then it should return the expected result while recording"

```

Comment:
Hmm. Actually, on a reread, I seem to have lost the "improvements" I was making here. Rewording.